sdk/contentsafety/Azure.AI.ContentSafety/README.md

```diff
@@ -0,0 +1,91 @@
+# Azure ContentSafety client library for .NET
+
+This section should give out brief introduction of the client library.
```

Comment:
Should this be filled out before being merged for the first time, or is another PR coming?

Reply:
This will be included in this PR, WIP.